praetorian_cli/sdk/entities/risks.py

```
@@ -279,59 +272,155 @@ def _hydrate_evidence(self, key):
+        return sorted(proof_files, key=sort_key)
+
+    def _get_file_content(self, file_record):
+        file_path = file_record.get('name', '')
+        if not file_path:
+            return ''
+
+        try:
+            return self.api.files.get_utf8(file_path)
+        except UnicodeDecodeError:
+            return self.api.files.get(file_path).decode('utf-8', errors='replace')
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)